content/physical_properties/physical_properties_density.rst

```diff
@@ -0,0 +1,61 @@
+.. _physical_properties_density:
+
+Density
+*******
+
+Some Definitions
+================
+
+For gravity survey, the physical property of concern is density. A densier buried object would create an higher attraction than the background. A hole, with a density of 0, would create a lower attraction than the background. Difference of densities in the subsurface can be mapped by measuring the variations of the earth gravity field.
```

Comment:
can you put the 0 in `\\( \\)` (makes it easier to recognize it as a number b/c then it renders as math